src/components/facets/FacetRow.tsx

```diff
@@ -52,15 +53,15 @@ export class FacetRow extends React.Component<IFacetRowProps, any> {
                     <button type='button' className={this.props.isChecked && this.isExclude ? 'exclude-box' : ''}>
                         <Svg svgName='clear' className='icon hide exclude-icon' svgClass='fill-red' />
                     </button>
-                    <span className='label'>{this.getLabel()}</span>
+                    {this.getLabel(<span className={labelClassName}>{this.props.facetRow.formattedName}</span>)}

```

Comment:
I find it weird to call a method getLabel but this method as a "label" parameter. Instead of using a parameter in the function you should create a const in the actual function:
```

private getLabel(): JSX.Element {
  const labelClassName = classNames({
    'text-exclude': this.props.isChecked && this.isExclude
  });
  const label = <span className={labelClassName}>{this.props.facetRow.formattedName}</span>;
  ...
```